internal/rootcoord/dml_channels.go

```diff
@@ -278,12 +283,10 @@ func (d *dmlChannels) broadcastMark(chanNames []string, pack *msgstream.MsgPack)
 
 func (d *dmlChannels) addChannels(names ...string) {
 	for _, name := range names {
-		v, ok := d.pool.Load(name)
-		if !ok {
-			log.Error("invalid channel name", zap.String("chanName", name))
-			panic("invalid channel name: " + name)
+		dms, err := d.getMsgStreamByName(name)
+		if err != nil {
+			continue
```

Comment:
add one error message here?

Reply:
the error msg has been add in the `getMsgStreamByName` method.
![image](https://github.com/milvus-io/milvus/assets/21985684/88f284db-985b-4a89-afa6-faa71f64b92f)